src/bindings/python/src/pyopenvino/utils/utils.cpp

```diff
@@ -24,7 +24,8 @@ using Version = ov::pass::Serialize::Version;
 namespace Common {
 namespace utils {
 
-PY_TYPE check_list_element_type(const py::list& list) {
+template <typename T>
+PY_TYPE check_list_element_type(const T& list) {
```

Comment:
```suggestion
// Checks element type of list and set
PY_TYPE check_container_element_type(const T& container) {
```
If this method is going to be used for `set`, it would be good to also change variables names

Reply:
Ok, done.